integration-tests/deployment/ccip/add_lane_test.go

```diff
@@ -2,31 +2,46 @@ package ccipdeployment
 
 import (
 	"testing"
+	"time"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/require"
 
 	"github.com/smartcontractkit/chainlink-testing-framework/lib/utils/testcontext"
+
 	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/offramp"
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
-// TestAddLane covers the workflow of adding a lane
-// between existing supported chains in CCIP.
+// TestAddLane covers the workflow of adding a lane between two chains and enabling it.
+// It also covers the case where the onRamp is disabled on the OffRamp contract initially and then enabled.
```

Comment:
Trying to understand the use case - we send a message on an onramp that is not enabled, and what's the logic behind that? Is it that enabling it on the offramp is the "flip the switch"? 

In practice, using the test router, don't think this would be the flow, but might be missing something.

Reply:
goal is to make sure that the plugin gracefully doesn't attempt to process disabled lanes. Although in the happy path we'd disable inbound first still good to test